gcp-observability/src/test/java/io/grpc/gcp/observability/logging/GcpLogSinkTest.java

```diff
@@ -98,32 +100,31 @@ public class GcpLogSinkTest {
       .putFields("rpc_id", Value.newBuilder().setStringValue(rpcId).build())
       .build();
   private Logging mockLogging;
+  private GcpLogSink spySink;
```

Comment:
Only some tests use `spySink` - others use a local `spySink2`  which looks a little odd. May be we can get rid of this class member and just use a local `spySink` in all tests based on the test's requirements?

Reply:
Done